ash/src/lib.rs

```diff
@@ -182,6 +190,7 @@ macro_rules! match_in_struct {
 #[cfg(test)]
 mod tests {
     use super::vk;
+    use alloc::vec::Vec;
     #[test]
     fn test_ptr_chains() {
         let mut variable_pointers = vk::PhysicalDeviceVariablePointerFeatures::default();
```

Comment:
The test code below also calls `vec![]`, I'm still surprised why I can run `cargo t` with and without the _default_ `std` feature _without_ importing `use alloc::vec`, qualifying `alloc::vec![]`, _nor_ `#[macro_use]`... :confused:

Reply:
Yeah, I think it really likes to give you std implicitly. Not sure what the best way to test this really is -- for hecs someone contributed a CI step that sets `--target x86_64-unknown-none`, which is probably reliable but seems like a big hammer.

Reply:
Not really interested to use this test as a `no_std` hammer, just curious why it works when it "shouldn't" :)